backend/connection.go

```diff
@@ -18,9 +18,12 @@ const (
 	// many lines to tail from.
 	bytesToLines int64 = 120
 
-	// defaultTailLines is the number of lines to tail by default if the value
-	// is not overriden.
-	defaultTailLines int64 = 30
+	// defaultTailLines is the number of lines to tail by default.
+	defaultTailLines int64 = 1000
```

Comment:
would have to test it out in production, for me, usually a few 100 lines are way more than enough. 1000 lines is overwhelming :)